admiral/cqc_functions.sqf

```diff
@@ -116,7 +116,6 @@ adm_cqc_fnc_spawnGarrisonGroupUnits = {
         private _position = selectRandom _possiblePositions;
         _possiblePositions = _possiblePositions - [_position];
         private _unit = [_building buildingPos _position, _group, _unitTemplate, _zoneTemplate, UNIT_TYPE_ARRAY select UNIT_TYPE_INF] call adm_cqc_fnc_placeMan;
-        _unit setVariable ["adm_unit_isCqc", true, true];
```

Comment:
I added this, it used to be used in our custom danger.fsm (which we binned because FSMs are slooow).

I don't use it in inhouse anywhere and it's not referenced in hull or admiral anywhere, can reinstate if required but I was trying to cut down on unused global variables.

Reply:
Keep or remove @kami- ?

If keep can we at least make it a local variable as all AI is generally server owned so clients don't need access to the variable.

Reply:
Sorry, you can remove it.